src/api/batch/batches.md

```diff
@@ -64,11 +64,11 @@ The following table describes the Batch Types supported for loading.
     </tr>
   </thead>
   <tbody>
-    {%- assign batch_types = site.html_pages | where_exp:"item", "item.permalink contains '/api/batch-types/'" -%}
+    {%- assign batch_types = site.html_pages | where_exp:"item", "item.batch_type != nil" -%}
     {%- for batch_type in batch_types -%}
     <tr>
       <td>
-        <a href="{{ batch_type.url | absolute_url }}">{{ batch_type.type }}</a>
+        <a href="{{ batch_type.url | absolute_url }}">{{ batch_type.batch_type }}</a>
```

Comment:
So we're iterating batch_types to get batch_type.batch_type, maybe we could use other language here?

```
     {%- assign batch_pages = ... -%}
     {%- for page in batch_pages -%}
         <a href="{{ page.url | absolute_url }}">{{ page.batch_type }}</a>
```

Reply:
@heymatthew yeah it's a batch type page. Could rename to `page.batch_type`.